etc/motions/stand_up_back_dortmund_2022.json

```diff
@@ -408,7 +410,14 @@
               "ankle_roll": 0.017453292519943295
             }
           }
-        },
+        }
+      ]
+    },
+    {
+      "entry_condition": {
+        "FallenAbort": {}
+      },
```

Comment:
Does this work?

```suggestion
      "entry_condition": "FallenAbort",
```

Reply:
No, unfortunately not:
`invalid type: unit variant, expected newtype variant at line 417 column 39`